apps/t_shirt_delivery/templates/registration/change_tshirt.html

```diff
@@ -24,72 +24,72 @@
 
 
 <script>
-    let unique_years = [];
-    let years_and_options = {};
-    let years_and_tshirt_url = {}
-    let selected_year = null
-    let current_button = null
+    let uniqueYears = [];
```

Comment:
We do not want all of these variables to be available from global content, please wrap your code in an anonymous function.

e.g.

```html
<script>
let test = "test";
</script>

<script>
console.log(test);
</script>


```